Team2-Signoff-CO2 & NO2 Monitoring Subsystem - Signoff.md

jaredhooker opened the pull request:
Requesting Signoff for CO2 & NO2 Monitoring Subsystem

jaredhooker committed:
Create CO2 & NO2 Monitoring Subsystem - Signoff.md

JesseTNRoberts commented on Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md:
```diff
@@ -0,0 +1,207 @@
+
+    dioxide present in the greenhouse.
+
+  - Must alert when above or below 400-2000ppm to ensure proper plant growth.
```
Based on what? Also what does alert mean?

JesseTNRoberts commented on Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md:
```diff
@@ -0,0 +1,207 @@
+
+    time and money on wiring, while also allowing the system to be scalable.
+
+  - The transfer of data between the sensors and the Arduino must be nearly
```
What does this mean?

JesseTNRoberts commented on Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md:
```diff
@@ -0,0 +1,207 @@
+
+    the atmosphere.
+
+  - BLE (Bluetooth Low Energy) is lower in cost than the classic Bluetooth as BLE is
```
And it's not compatible with many Bluetooth devices... probably ought to check into that.

JesseTNRoberts commented on Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md:
```diff
@@ -0,0 +1,207 @@
+
+    dioxide present in the greenhouse.
+
+  - Must alert when above Nitrogen Dioxide level exceeds 5 ppm.
```
Based on what?

Noah-Jones37 replied:
The CDC's guidelines, which is source 2.

JesseTNRoberts commented on Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md:
```diff
@@ -0,0 +1,207 @@
+
+  - Must alert when above Nitrogen Dioxide level exceeds 5 ppm.
+
+- **Silicon Conformal Coating**
```
Why?

JesseTNRoberts reviewed (commented):
The formatting of this document looks very bad. Address this.

Show all the pins of your devices. I don't think this is sufficient for the connections. Why is there a resistive divider to power a device?

jaredhooker committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jaredhooker committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jaredhooker committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

JesseTNRoberts reviewed (commented):
There's no analysis of the design showing that the circuit will turn on the system when needed and will have the required battery life. I don't see anything analyzing whether this will have the needed sensitivity.

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Updated Buildable Schematic 
Updated Microcontroller to Arduino Nano 33 IoT
Updated BoM

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Updated Resources

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Explained Scalability.

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Formatting and Numbering

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Made Template for numbered Analysis that match the Constraints

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jax-commits committed:
Delete Documentation/Images/CADCommunicationsSubsystem.jpg

jax-commits committed:
Add files via upload

Noah-Jones37 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Updated Schematic

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Updated MIKROE Analysis

barhoton42 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Formatting

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Deleted unnecessary word

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Fixed Equations

barhoton42 committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

jax-commits committed:
Merge branch 'main' of https://github.com/RealityHertz/Greenhouse-Project into Team2-Signoff-CO2&NO2Monitoring-2

jax-commits committed:
Update CO2 & NO2 Monitoring Subsystem - Signoff.md

Photo Messed up with merge

JesseTNRoberts reviewed (commented):
We discussed that the constraint regarding 400-2000ppm seems to be incorrect.

Noah-Jones37 commented:
> We discussed that the constraint regarding 400-2000ppm seems to be incorrect.

After further research, we concluded that the values are correct. A source has been cited to prove this.

jax-commits committed:
Minutes for 11-10-2023 Added.

jax-commits committed:
Merge branch 'Team2-Signoff-CO2&NO2Monitoring-2' of https://github.com/RealityHertz/Greenhouse-Project into Team2-Signoff-CO2&NO2Monitoring-2

jax-commits committed:
Minutes Removed.

jax-commits committed:
Minutes for 11-10-2023 Added.

jax-commits committed:
Delete Documentation/Meeting Minutes/Minutes - 11-10-2023.md

jax-commits committed:
Removed Image.

jax-commits committed:
Image for Lighting Added.
